tdvt/samples/connector-tests/propertiesbuilder.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version='1.0' encoding='utf-8' ?>
+<testCases>
+    <testCase>
+        <connection authentication='username-password' class='postgres_odbc' dbname='TestV1' odbc-native-protocol='' one-time-sql='' port='5432' server='postgres' username='test' password='pswd' />
```

Comment:
This would only apply to postgres_jdbc connector

Reply:
Added a comment for this test to only be used for jdbc based connectors and updated class to postgres_jdbc